src/composables/useGlobalErrorHook.ts

```diff
@@ -5,7 +5,11 @@ export function useGlobalErrorHook() {
   const messageStore = useMessageStore();
 
   const onError = (event: ErrorEvent) => {
-    messageStore.addError('Application error (click for details)', event.error);
+    console.error(event);
```

Comment:
Good!  Thinking we should send more errors to Sentry via:
`Sentry.captureException` https://docs.sentry.io/platforms/javascript/usage/

or 
```
Sentry.init({
  dsn: 'https://your-sentry-server-dsn',
  integrations: [
    new CaptureConsole({
      levels: ['error']
    })
  ],
...
```

Reply:
I like this idea! I'll cover this in a later PR.